Source/WebKit/UIProcess/API/gtk/WebKitUserContentManager.h

```diff
@@ -118,6 +118,19 @@ webkit_user_content_manager_remove_filter_by_id                        (WebKitUs
 WEBKIT_API void
 webkit_user_content_manager_remove_all_filters                         (WebKitUserContentManager *manager);
 
+#define WEBKIT_TYPE_USER_SCRIPT_MESSAGE    (webkit_user_script_message_get_type())
+
+typedef struct _WebKitUserScriptMessage WebKitUserScriptMessage;
+
+WEBKIT_API GType
+webkit_user_script_message_get_type       (void);
+
+WEBKIT_API WebKitUserScriptMessage *
+webkit_user_script_message_ref            (WebKitUserScriptMessage    *user_script_message);
+
+WEBKIT_API void
+webkit_user_script_message_unref          (WebKitUserScriptMessage    *user_script_message);
+
```

Comment:
The headers are missing the declarations for `webkit_user_script_message_get_message()`, `webkit_user_script_message_reply()`, and `webkit_user_script_message_error()`.